src/dune

```diff
@@ -1,7 +1,7 @@
 (executable
  (name gencode)
  (modules gencode)
- (libraries lambdasoup uutf))
+ (libraries uutf))
```

Comment:
@Swrup did you remove lambdasoup from dune-project and regenerated the opam file ?